theme/styles/_config.scss

```diff
@@ -1,5 +1,6 @@
 $colors: (
   'black': #000000,
+  'yellow': '#fff84f',
```

Comment:
I think you want to scrap the quotes on the yellow value

Reply:
!!! thanks!